src/test/java/org/dita/dost/IntegrationTestPdf.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * This file is part of the DITA Open Toolkit project.
+ *
+ * Copyright 2018 Jarno Elovirta
+ *
+ * See the accompanying LICENSE file for applicable license.
+ */
+
+package org.dita.dost;
+
+import org.junit.Test;
+
+import java.io.File;
+import java.nio.file.Paths;
+
+import static org.dita.dost.AbstractIntegrationTest.Transtype.HTML5;
+import static org.dita.dost.AbstractIntegrationTest.Transtype.PDF;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class IntegrationTestPdf extends AbstractIntegrationTest {
+
+    public AbstractIntegrationTest builder() {
+        return new IntegrationTestPdf();
+    }
+
+    @Override
+    Transtype getTranstype(Transtype transtype) {
+        return transtype;
+    }
+
+    @Test
+    public void pdf() throws Throwable {
```

Comment:
This just tests that the generated FO file matches the expected file. So at least there is some test that fails when you change the FO output. The test case is suboptimal and should at some point be replaced with a test that has all DITA elements. We could then use that for HTML5 too.